wd_core.au3

```diff
@@ -113,10 +113,11 @@ Global Enum _
 		$_WD_ERROR_UserAbort, _ ;
 		$_WD_ERROR_FileIssue, _ ;
 		$_WD_ERROR_NotSupported, _ ;
-		$_WD_ERROR_AlreadyDefined, _ ;
+		$_WD_ERROR_AlreadyDefined, _ ; Used in _WD_CapabilitiesDefine and __WD_CapabilitiesInitialize
+		$_WD_ERROR_CapabilityCaseSensivity, _ ; Used in _WD_CapabilitiesAdd
```

Comment:
This seems unnecessary to me. Please take a look at implementing this without adding another error code.

Reply:
I used it here:
```auotit
	ElseIf _
			StringRegExp($key, '(?i)' & $_WD_KEYS__STANDARD_PRIMITIVE, $STR_REGEXPMATCH) Or _
			StringRegExp($key, '(?i)' & $_WD_KEYS__STANDARD_OBJECT, $STR_REGEXPMATCH) Or _
			StringRegExp($key, '(?i)' & $_WD_KEYS__STANDARD_OBJECT_ARRAY, $STR_REGEXPMATCH) Or _
			StringRegExp($key, '(?i)' & $_WD_KEYS__SPECIFICVENDOR_PRIMITIVE, $STR_REGEXPMATCH) Or _
			StringRegExp($key, '(?i)' & $_WD_KEYS__SPECIFICVENDOR_ARRAY, $STR_REGEXPMATCH) Or _
			StringRegExp($key, '(?i)' & $_WD_KEYS__SPECIFICVENDOR_OBJECT, $STR_REGEXPMATCH) _
			Then
		If $_WD_ERROR_MSGBOX Then MsgBox($MB_ICONERROR + $MB_OK + $MB_TOPMOST, @ScriptLineNumber, _
				'Capability name are case sensitive.' & @CRLF & _
				'Check proper case in $_WD_KEYS__*** for:' & @CRLF & _
				'"' & $key & '"')
		Return SetError(__WD_Error($sFuncName, $_WD_ERROR_CapabilityCaseSensivity, 'Capability name are case sensitive ( check proper case in $_WD_KEYS__*** for "' & $key & '"). ' & $s_Parameters_Info))
	Else
		Return SetError(__WD_Error($sFuncName, $_WD_ERROR_NotSupported, 'Not supported KEY parameter ( must be defined in $_WD_KEYS__*** ). ' & $s_Parameters_Info))
	EndIf
```

IMO it is different case and need different ERROR CODE.

Reply:
There are 20 existing error codes, and I imagine you can find one that can work in this situation. IMO, you can still use $_WD_ERROR_NotSupported and simply adjust the error message passed to __WD_Error.

Reply:
changed:

[core: $_WD_ERROR_CapabilityCaseSensivity removal](https://github.com/Danp2/au3WebDriver/pull/285/commits/fccfd244fbc6bcde2a1bc4287ef7277b0ef2c4fc)

[$_WD_ERROR_CapabilityCaseSensivity > NotSupported](https://github.com/Danp2/au3WebDriver/pull/285/commits/643191938cb0d90bc9abbe046d20338614c7aa50)